controllers/feeds/getFeedsV2.js

```diff
@@ -19,10 +19,11 @@ const {
   modifyReactionsPost,
   modifyFeedIsFollowingTarget
 } = require('../../utils/post');
-const {DomainPage, Locations, User} = require('../../databases/models');
+const {DomainPage, Locations, User, UserFollowUser} = require('../../databases/models');
 const RedisDomainHelper = require('../../services/redis/helper/RedisDomainHelper');
 const {ACTIVITY_THRESHOLD} = require('../../config/constant');
 const UsersFunction = require('../../databases/functions/users');
+const UserFollowUserFunction = require('../../databases/functions/userFollowUser');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new imports `UserFollowUser` and `UserFollowUserFunction` are added correctly. However, it's important to ensure that these modules are properly exported from their respective files and they do not have any side effects that could affect the current module.
```diff
- const {DomainPage, Locations, User} = require('../../databases/models');
+ const {DomainPage, Locations, User, UserFollowUser} = require('../../databases/models');
  const RedisDomainHelper = require('../../services/redis/helper/RedisDomainHelper');
  const {ACTIVITY_THRESHOLD} = require('../../config/constant');
  const UsersFunction = require('../../databases/functions/users');
+ const UserFollowUserFunction = require('../../databases/functions/userFollowUser');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->